docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerConfigurator.java

```diff
@@ -61,18 +64,22 @@ public void configure(@Observes CubeConfiguration event, ArquillianDescriptor ar
     }
 
     private void configure(ArquillianDescriptor arquillianDescriptor) {
-        operatingSystemFamilyInstanceProducer.set(new OperatingSystemResolver().currentOperatingSystem().getFamily());
-        Map<String, String> config = arquillianDescriptor.extension(EXTENSION_NAME).getExtensionProperties();
-        CubeDockerConfigurationResolver resolver = new CubeDockerConfigurationResolver(topInstance.get(),
-            dockerMachineInstance.get(),
-            boot2DockerInstance.get(),
-            operatingSystemFamilyInstanceProducer.get());
-        resolver.resolve(config);
-        CubeDockerConfiguration cubeConfiguration = CubeDockerConfiguration.fromMap(config, injectorInstance.get());
-        cubeConfiguration = resolveDynamicNames(cubeConfiguration);
-        System.out.println(cubeConfiguration);
-        hostUriContextInstanceProducer.set(new HostIpContext(cubeConfiguration.getDockerServerIp()));
-        configurationProducer.set(cubeConfiguration);
+        try {
+            operatingSystemFamilyInstanceProducer.set(new OperatingSystemResolver().currentOperatingSystem().getFamily());
+            Map<String, String> config = arquillianDescriptor.extension(EXTENSION_NAME).getExtensionProperties();
+            CubeDockerConfigurationResolver resolver = new CubeDockerConfigurationResolver(topInstance.get(),
+                dockerMachineInstance.get(),
+                boot2DockerInstance.get(),
+                operatingSystemFamilyInstanceProducer.get());
+            resolver.resolve(config);
+            CubeDockerConfiguration cubeConfiguration = CubeDockerConfiguration.fromMap(config, injectorInstance.get());
+            cubeConfiguration = resolveDynamicNames(cubeConfiguration);
+            System.out.println(cubeConfiguration);
+            hostUriContextInstanceProducer.set(new HostIpContext(cubeConfiguration.getDockerServerIp()));
+            configurationProducer.set(cubeConfiguration);
+        } catch (ExecutionException e) {
+            log.log(Level.WARNING, "Error executing docker command", e);
```

Comment:
Maybe this should be `SEVERE` since you cannot run anything isn't it? Also at then end should throw a Runtime exception to not try to execute this test. Without stopping it, the test is not aborted so a test failure about cannot find docker host would be thrown.